dev-aws/kafka-shared/iam_credentials.tf

```diff
@@ -0,0 +1,35 @@
+resource "kafka_topic" "iam_credentials_v1" {
+  name               = "auth-customer.iam-credentials-v1"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    # retain 100MB on each partition
+    "retention.bytes" = "104857600"
+    # keep data for 7 days
+    "getention.ms" = "604800000"
+    # allow max 1 MB for a message
+    "max.message.bytes" = "1048576"
+    "compression.type"  = "zstd"
+    "cleanup.policy"    = "delete"
+  }
+}
+
+resource "kafka_acl" "iam_credentials_v1_access" {
```

Comment:
I think it would be good to start using the `producer` module. See customer billing example: https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/dev-aws/kafka-shared/customer-billing.tf#L15-L31

I have a task to migrate it all to the modules, but didn't get to it yet

Reply:
Nice, seems clear. Pls have a look